src/super_gradients/recipes/quantization_params/default_tensorrt_ptq.yaml

```diff
@@ -0,0 +1,34 @@
+quantizer:
+  TRTPTQQuantizer:
+    selective_quantizer_params:
+      calibrator_w: "max"        # calibrator type for weights, acceptable types are ["max", "histogram"]
+      calibrator_i: "histogram"  # calibrator type for inputs acceptable types are ["max", "histogram"]
+      per_channel: True          # per-channel quantization of weights, activations stay per-tensor by default
+      learn_amax: False          # enable learnable amax in all TensorQuantizers using straight-through estimator
+      skip_modules: []
+
+    calib_params:
+      histogram_calib_method: "percentile"  # calibration method for all "histogram" calibrators, acceptable types are ["percentile", "entropy", "mse"], "max" calibrators always use "max"
+      percentile: 99.99                     # percentile for all histogram calibrators with method "percentile", other calibrators are not affected
+      num_calib_batches: 128                # number of batches to use for calibration, if None, 512 / batch_size will be used
+      verbose: False                        # if calibrator should be verbose
+
+exporter: ONNXExporter
+
+output_path: ${architecture}_trt_ptq_int8.onnx
+
+# Model-specific parameters
+export_params:
+    preprocessing: True
+    postprocessing: True
+
+    batch_size: 1
+    confidence_threshold: 0.2
+    onnx_simplify: True
+    onnx_export_kwargs: {}
+
+    detection_nms_iou_threshold: 0.5
+    detection_max_predictions_per_image: 128
+    detection_num_pre_nms_predictions: 1000
+    detection_predictions_format: flat
+    detection_postprocessing_use_tensorrt_nms: False
```

Comment:
I find it pretty confusing that we have an Exporter object, but model specific export_params....
Especially given the fact that there's detection_postprocessing_use_tensorrt_nms argument here.

Reply:
This is nothing new btw, ExportParams are not introduced in this PR, it's just I've added explicitly all the fields that exists in ExportParams structure to help user start editing these parameters.
I can duplicate docstrings on each setting, but it's going quite long YAML (see [run_sanity_tests_flow](https://github.com/Deci-AI/super-gradients/blob/cbc670f6f3a3bcb5b79926478f182b466feddf2f/src/super_gradients/conversion/export_params.py#L8))